src/Testing/FactoryBuilder.php

```diff
@@ -213,18 +213,18 @@ protected function callClosureAttributes(array $attributes)
         return array_map(function ($attribute) use ($attributes) {
             if ($attribute instanceof \Closure) {
                 $entity = $attribute($attributes);
-                if (is_object($entity)) {
+	            if (is_iterable($entity)) {
```

Comment:
This is a php7.1 method, but the current branch also still allows 7.0

Perhaps we can do is_array || isntanceof Traversable , as that's basically what is_iterable does ?

Reply:
Fixed.  I always forget iterable was introduced in 7.1 and not 7.0.